src/jetson/models/SSD/ssd.py

```diff
@@ -194,13 +199,24 @@ def multibox(vgg, extra_layers, cfg, num_classes):
     '300': [256, 'S', 512, 128, 'S', 256, 128, 256, 128, 256],
     '512': [],
 }
+
+# number of boxes per feature map location
 mbox = {
-    '300': [4, 6, 6, 6, 4, 4],  # number of boxes per feature map location
+    '300': [4, 6, 6, 6, 4, 4],
     '512': [],
 }
 
```

Comment:
why are the 2 above dicts globals of hardcoded values? This seems like something that could be abstracted into a possible configuration file especially if they are constant throughout the program